src/ui/profile_preferences.ui

```diff
@@ -2423,10 +2423,54 @@
             </property>
            </widget>
           </item>
+          <item row="1" column="0" alignment="Qt::AlignRight">
+           <widget class="QLabel" name="label">
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+              <horstretch>1</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="text">
+             <string>Suppress non-error debug output from Timers with interval less than (h:mm:ss.zzz):</string>
```

Comment:
How about "Show debug info on OK Timers only when their interval is more than" ...? Getting rid of multiple negations here :)